oak_containers_system_image/base_image/build-base.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+
+cd "$SCRIPTS_DIR/.."
+
+# Fix the file permissions that will be loaded into the system image, as Git doesn't track them.
+# Unfortunately we can't do it in Dockerfile (with `COPY --chown`), as that requires BuildKit.
+chmod --recursive a+rX files/
+
+docker build . --tag=oak-containers-sysimage-base:latest -f base_image/Dockerfile
```

Comment:
It is a bit odd that the Dockerfile is in a different folder than the context. Would it be possible to have everything at the same level (so that e.g. the `COPY` instructions are relative to the Dockerfile folder)?

Reply:
I was trying to organize the files so that base-image related files were grouped together.

But I can't reference files from parent directories in the Dockerfile, so I set the script up to run from the parent, instead.

I can just remove this subdirectory completely and keep everything in the same directory, if that's preferable.